### PR TITLE
repos: update for kirkstone branch

### PR DESCRIPTION
**Pull request by dylan-matthew-turner**
We will want to perform upstream merges for kirkstone using the upstream merge script. However, for this to work, repos.conf needs to be set up to point to the kirkstone branches of upstreams

### CODE REVIEW

**Review by chaitu236 — APPROVED** [review]

File: scripts/dev/upstream_merge/repos.conf [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+sources/openembedded-core    https://git.openembedded.org/openembedded-core        kirkstone             nilrt/master/kirkstone
+sources/meta-virtualization  https://git.yoctoproject.org/meta-virtualization      kirkstone             nilrt/master/kirkstone
+sources/meta-security        https://git.yoctoproject.org/meta-security            kirkstone             nilrt/master/kirkstone
+sources/meta-sdr             https://github.com/balister/meta-sdr                  kirkstone             nilrt/master/kirkstone
+sources/meta-qt5             https://github.com/meta-qt5/meta-qt5                  kirkstone             nilrt/master/kirkstone
+sources/meta-qt5-extra       https://github.com/schnitzeltony/meta-qt5-extra       kirkstone             nilrt/master/kirkstone
+sources/meta-rauc            https://github.com/rauc/meta-rauc                     kirkstone             nilrt/master/kirkstone
+sources/pyrex                https://github.com/garmin/pyrex                       master                nilrt/master/kirkstone
+sources/meta-mingw           https://git.yoctoproject.org/meta-mingw               kirkstone             nilrt/master/kirkstone
```
Comment (chaitu236) [inline scripts/dev/upstream_merge/repos.conf:14]: We seem to have [removed](https://github.com/ni/nilrt/commit/68a63c866b9de1b4fd23ce36306e7114f29c6945) `meta-mingw` layer from hardknott and [added it back](https://github.com/ni/nilrt/commit/031b41a6a392b03f54e141c902144b21ceed82c0) but we don't have it listed in hardknott's `repos.conf`. Do you mind adding it?

Comment (dylan-matthew-turner) [reply scripts/dev/upstream_merge/repos.conf:14]: Can do

Comment (dylan-matthew-turner) [reply scripts/dev/upstream_merge/repos.conf:14]: https://github.com/ni/nilrt/pull/208

Comment (dylan-matthew-turner) [reply scripts/dev/upstream_merge/repos.conf:14]: The local branch for meta-mingw appears to be wrong according to the upstream merge script. I'll correct it

Comment (dylan-matthew-turner) [reply scripts/dev/upstream_merge/repos.conf:14]: https://github.com/ni/nilrt/pull/209

**Review by amstewart — CHANGES_REQUESTED** [review]
[`s/kirstone/kirkstone/`](https://github.com/ni/nilrt/pull/207/commits/d3a1e03f38f2df501aec23155fadfd4e0f427161)

**Review by amstewart — APPROVED** [review]

**Pull request merged**